libpod/container_config.go

```diff
@@ -84,6 +84,10 @@ type ContainerConfig struct {
 	// This field should never be written to the db, the json tag ensures this.
 	rewrite bool `json:"-"`
 
+	// MountAllDevices is an option to indicate whether a privileged container
+	// will mount all the host's devices
+	MountAllDevices bool `json:"mountAllDevices"`
```

Comment:
Can you move this into ContainerMiscConfig? The structs are embedded, so no further changes should be required.